src/app/modules/main/profile_section/profile/controller.nim

```diff
@@ -61,7 +61,11 @@ method getProfile*[T](self: Controller[T]): item.Item =
   return item
 
 method storeIdentityImage*[T](self: Controller[T], address: string, image: string, aX: int, aY: int, bX: int, bY: int): identity_image.IdentityImage =
-  self.profileService.storeIdentityImage(address, image, aX, aY, bX, bY)
+  result = self.profileService.storeIdentityImage(address, image, aX, aY, bX, bY)
+  singletonInstance.userProfile.setThumbnailImage(result.thumbnail)
+  singletonInstance.userProfile.setLargeImage(result.large)
 
 method deleteIdentityImage*[T](self: Controller[T], address: string): string =
-  self.profileService.deleteIdentityImage(address)
\ No newline at end of file
+  result = self.profileService.deleteIdentityImage(address)
```

Comment:
@Khushboo-dev-cpp this is a response of `deleteIdentityImage` call which currently returns an empty string, but since that is a response of delete command, that is not something what will be here in the future or that method in the future may return success or failor of deleting image action.  So I would suggest using just this and not relaying on the response:
```
singletonInstance.userProfile.setThumbnailImage("")
singletonInstance.userProfile.setLargeImage("")
```

Reply:
Done!